site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -523,10 +527,13 @@ function printPreview(target){
 						<td>$validator.escapeHtml($group.name)</td>
 						<td>
 							#if ($allowUpdateSiteMembership)
-							<form name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
+							<form id="removeGroupForm" name="removeGroupForm" class="inlineForm" method="post" action="#toolForm("SiteAction")">
```

Comment:
You shouldn't have multiple elements in the same page with the same ID (as this is in a loop).